client/layout/index.js

```diff
@@ -63,12 +63,13 @@ class Layout extends Component {
 		const { isEmbeded, ...restProps } = this.props;
 		return (
 			<div className="woocommerce-layout">
-				{ window.wcAdminFeatures[ 'activity-panels' ] && <Slot name="header" /> }
+				<Slot name="header" />
 				<TransientNotices />
 
 				<div className="woocommerce-layout__primary" id="woocommerce-layout__primary">
 					{ window.wcAdminFeatures[ 'store-alerts' ] && <StoreAlerts /> }
-
+					{ window.wcAdminFeatures.onboarding &&
+						isEmbeded && <strong>Testing. @todo Remove this before merging.</strong> }
```

Comment:
Adding this as an example of a piece of code behind a feature flag, that would get exposed in core on an existing WooCommerce page.